monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties

```diff
@@ -1,2 +1,2 @@
-GLOBAL_DISCOUNT=0
+GLOBAL_DISCOUNT=20
```

Comment:
Per User Story, I thought we said 25%! Let's go with 25.

Reply:
Are you 100% sue?  I talked to Joe N. and i thought they changed rqmts?

Reply:
This is not what we signed up for!